libcontainer/configs/validate/validator.go

```diff
@@ -154,16 +161,26 @@ func (v *ConfigValidator) sysctl(config *configs.Config) error {
 			}
 		}
 		if strings.HasPrefix(s, "net.") {
-			if config.Namespaces.Contains(configs.NEWNET) {
-				if path := config.Namespaces.PathOf(configs.NEWNET); path != "" {
-					if err := checkHostNs(s, path); err != nil {
-						return err
-					}
+			// Is container using host netns?
+			netOnce.Do(func() {
+				if !config.Namespaces.Contains(configs.NEWNET) {
+					hostnet = true
+					return
 				}
-				continue
-			} else {
-				return fmt.Errorf("sysctl %q is not allowed in the hosts network namespace", s)
+				path := config.Namespaces.PathOf(configs.NEWNET)
+				if path == "" {
+					// hostnet = false
```

Comment:
Shouldn't hostnet be true if path is empty?

Reply:
Never mind. This is fine :+1:

Reply:
yeah, this is the case when configs.NEWNET is set but the path is not (which means "make a new netns").

Reply:
I have added a tiny clarification to that comment